src/schema.js

```diff
@@ -1,13 +1,90 @@
-var { buildSchema } = require('graphql');
+var {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+  GraphQLSchema,
+  GraphQLList,
+  GraphQLNonNull
+} = require('graphql');
+
+// Import the Database connection.
+var Db = require('./db')
+
+// Define the GraphQL types for the schema.
+// Eventually these might include things like Pages, Questions, Answers etc.
+var Message = new GraphQLObjectType({
```

Comment:
The convention in JS is to use uppercase for class definitions and lowercase for instances, eg:

```js
var message = new GraphQLObjectType
```